htdocs/product/stock/stockatdate.php

```diff
@@ -245,12 +245,12 @@
 
 $sql = 'SELECT p.rowid, p.ref, p.label, p.description, p.price, p.pmp,';
 $sql .= ' p.price_ttc, p.price_base_type, p.fk_product_type, p.desiredstock, p.seuil_stock_alerte,';
-$sql .= ' p.tms as datem, p.duration, p.tobuy, p.stock, ';
+$sql .= ' p.tms as datem, p.duration, p.tobuy, p.stock, p.pmp, ';
```

Comment:
Field p.pmp seems already added 2 lines before...

Reply:
Hi @eldy 
I will review the code.

Reply:
Hi @Eldy
Sorry for having been so late.

In version 14.0, I just reviewed the latest code and there is no p.pmp in line 246 (or 245 to be exact).
I will anyway make the same change as seen in v17 or even v19 if you want.

I will also submit  a new PR on develop to change the estimatedvalue to currentvalue in the SQL request to clear the code.
Let me know your decision.